nuxeo-features/nuxeo-search/nuxeo-search-rest-api/src/test/resources/pageprovider-test-contrib.xml

```diff
@@ -98,7 +98,6 @@
       <pattern>
         SELECT * FROM Document where dc:title=:foo
       </pattern>
-      <sort column="dc:title" ascending="true" />
```

Comment:
To have `SearchTest.iCanPerformPageProviderWithNamedParametersInvalid` and `SearchTest.iCanPerformPageProviderWithNamedParametersAndDocInvalid` test a different error message (with a default sort and without).